### PR TITLE
Rename test data factories and refactoring

### PR DESCRIPTION
**Pull request by AlexanderLitus**
See the latest commits.

### CODE REVIEW

File: core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java [hunk 1]
```diff
@@ -83,7 +84,7 @@ public void save_and_read_several_messages() {
 
     private void testSaveAndReadMessage(String messageId) {
 
-        final ProjectId expected = newProjectId(messageId);
+        final ProjectId expected = TestAggregateIdFactory.createProjectId(messageId);
```
Comment (mikhailmikhaylov) [inline core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java]: Remove "TestAggregateIdFactory" as this method is already static-imported

File: core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java [hunk 2]
```diff
@@ -59,7 +60,7 @@ public void return_empty_message_if_read_one_record_by_null_id() {
     @Test(expected = NullPointerException.class)
     public void throw_exception_if_write_by_null_id() {
         //noinspection ConstantConditions
-        storage.write(null, newProjectId("testId"));
+        storage.write(null, TestAggregateIdFactory.createProjectId("testId"));
```
Comment (mikhailmikhaylov) [inline core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java]: Same here

File: core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java (hunk 1 above)
Comment (AlexanderLitus) [inline core-java/src/test/java/org/spine3/server/storage/EntityStorageShould.java]: Done.

**Comment by mikhailmikhaylov** [comment]
Looks good for me

**Pull request merged**